fvm/fvm_test.go

```diff
@@ -10,6 +10,8 @@ import (
 	"testing"
 
 	stdlib2 "github.com/onflow/cadence/stdlib"
+	flow2 "github.com/onflow/flow-go-sdk"
```

Comment:
```suggestion
	flowsdk "github.com/onflow/flow-go-sdk"
```

Reply:
Cleaned up this and related import aliases in 0abd8bec2c389f1a44c98b7c232f1f7e1feedb49